Gemfile

```diff
@@ -3,7 +3,9 @@
 source 'https://rubygems.org'
 gem 'cucumber'
 gem 'dotenv-rails', groups: %i[development test]
+gem 'faker'
```

Comment:
Just a suggestion. You could use the Ruby's [pessimistic operator](https://thoughtbot.com/blog/rubys-pessimistic-operator) for the gems you are adding from now on.